backend/lib/use-cases/validations.ts

```diff
@@ -0,0 +1,43 @@
+import { filterToExtendedAscii, isValidUserInput } from '../../../common/src/cams/sanitization';
+import { BadRequestError } from '../common-errors/bad-request';
+
+export function sanitizeDeep<T>(input: T, moduleName: string): T {
+  const seen = new WeakSet();
+  const MAX_OBJECT_DEPTH = parseInt(process.env.MAX_OBJECT_DEPTH) ?? 50;
+  const MAX_OBJECT_KEY_COUNT = parseInt(process.env.MAX_OBJECT_KEY_COUNT) ?? 1000;
```

Comment:
We know we have some large objects. If we want to log the depth and key counts we see to get an idea for how large these numbers should really be we can do that, but we don't want to break the app by going too small.
```suggestion
  const MAX_OBJECT_KEY_COUNT = parseInt(process.env.MAX_OBJECT_KEY_COUNT) ?? 15000;
```